### ISSUE TITLE
Add support to accept only email addresses

### ISSUE THREAD

**Issue by vkbytes** [mdelobelle/metadatamenu#197]
It will good to have an option to only accept email address as values. This can be added as an option

**Comment by simonausten** [comment]
Regex validation on the input would be great (and would cover this, as well as URLs etc...)

**Comment by mdelobelle** [comment]
Good idea. I think I could add a regex validator to input fields with a list of most common use cases
